docs/docs/getting-started/getting-started.mdx

```diff
@@ -90,7 +90,7 @@ The Groundlight system is designed to provide consistent, highly confident answe
 scenarios where you need reliable visual detection.
 
 Let's see how to use Groundlight to analyze an image:
-```python title="ask.py"
```

Comment:
will comeback and fix this the issue is that when running on GH actions youtube identifies this as a bot.